docs/Developers/Review.rst

```diff
@@ -3,19 +3,32 @@ Reviewing
 
 This document is a guide to coala's review process.
 
+Am I Good Enough to Do Code Review?
+-----------------------------------
+
+**Yes,** if you already fixed a newcomer issue.
+
+Please review at least one Pull Request for every review you receive. Start
+with other people's newcomer issues and advance as you do with your own issues.
```

Comment:
"advance as you do with your own issues" is a little confusing and could use some rewording imo